spec/Interoperation.tex

```diff
@@ -257,10 +265,30 @@ \subsubsection{Referring to External C Structs}
 these structs but Chapel cannot guarantee the contents or memory story of
 fields of which it has no knowledge.
 
+If the optional \sntx{external-name} is supplied, then it is used verbatim as
+the exported struct symbol.
+
 A C header file containing the struct's definition in C must be specified on the
-chpl compiler command line.  Note that currently only typdef'd C structures are
-supported.  That is, in the C header file, the \chpl{struct} must be supplied
-with a type name through a \chpl{typedef} declaration.
+chpl compiler command line.  Note that only typdef'd C structures are supported
+by default.  That is, in the C header file, the \chpl{struct} must be supplied
+with a type name through a \chpl{typedef} declaration. If this is not true, you
+can use the \sntx{external-name} part to apply the \chpl{struct} specifier.
+As an example of this, given a C declaration of:
+
+\begin{chapel}
+  struct Vec3 {
+    double x, y, z;
+  };
+\end{chapel}
+
+in Chapel you would refer to this \chpl{struct} via
+
+\begin{chapel}
+  extern "struct Vec3" record Vec3 {
+    var x, y, z: real(64);
+  }
+\end{chapel}
```

Comment:
There are no testable examples in `Interoperation.tex`. I was assuming that our spec tests are not fancy enough to be able to provide C headers/code to a chapel program.